ai-plan/public/ai-plan-governance/traces/ai-plan-governance-trace.md

```
@@ -68,3 +68,28 @@
+- 将旧 `local-plan` 中分散的五个阶段计划整合进主题内历史跟踪归档，并额外补写一份基于 todo 基线整理出的历史 trace，
+  显式记录“缺少原始 trace，只能恢复稳定结论”的边界
+- 新建精简版 active tracking / trace 入口，只保留当前恢复点、活跃事实、风险与下一步
+- 在 `ai-plan/public/README.md` 中建立
+  `feat/coroutine-optimization` -> `coroutine-optimization` 的 worktree 映射，并把 `ai-plan-governance` 作为 secondary topic 保留
+- 删除旧 `local-plan` 目录，验证当前工作树根目录已不再保留 legacy 私有恢复入口
+- 额外完成验证：
+  - `find ai-plan/public/coroutine-optimization -maxdepth 3 -type f | sort`
+  - `test ! -e local-plan`
+  - `dotnet build GFramework.Core.Abstractions/GFramework.Core.Abstractions.csproj -c Release -p:RestoreFallbackFolders=`
+
+### 下一步
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)